packages/compass-sidebar/src/components/non-genuine-marker.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+
+import {
+  Badge,
+  BadgeVariant,
+  Icon,
+  css,
+  spacing,
+  useFocusRing,
+} from '@mongodb-js/compass-components';
+
+const nonGenuineMarkerContainer = css({
+  padding: `0 ${spacing[2]}px ${spacing[2]}px`,
+});
+
+const nonGenuineMarkerButton = css({
+  background: 'inherit',
+  padding: 0,
+  margin: 0,
+  border: 'none',
+  cursor: 'pointer',
+});
+
+export default function NonGenuineMarker({
+  isGenuine,
+  showNonGenuineModal,
+}: {
+  isGenuine?: boolean;
+  showNonGenuineModal: () => void;
+}) {
+  const focusRingProps = useFocusRing<HTMLButtonElement>();
+
+  // isGenuine === undefined means we haven't loaded the info yet
+  if (isGenuine !== false) {
+    return null;
+  }
+
+  return (
+    <div className={nonGenuineMarkerContainer}>
+      <button
+        {...focusRingProps}
+        type="button"
```

Comment:
There was this weird thing where focusRingProps brought type="button" which then made typescript complain because button is the string "button" and not the literal "button". Which is annoying because type is optional here and the default is fine. Not sure how best/where to fix that but I worked around it by just having type="button" after the focus ring props so that I can specify it again as the literal.

Let me know if there's a better fix.

Reply:
Yeah, this feels a bit like a bug in the TS support for JSX/React. 
```suggestion
        type={focusRingProps.type as 'button'}
```

might be a bit better, it keeps the logic of `{...focusRingProps}` and makes it clear that this line is there to silence TS, not because it is actually relevant logic?